### PR TITLE
Update README with CI and Trello info

### PR DESCRIPTION
**Pull request by kasia251011**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
-# Getting Started with Create React App
+# Exchange rate project 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Purpose of this project is to display statistics of excange rates.
```
Comment (cpprian) [inline README.md:3]: small typo

**Review by cpprian — APPROVED** [review]
There is a small typo, rest lgfm

File: README.md [hunk 2]
```diff
@@ -14,33 +21,10 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
...
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
+## Continous Integration
+CI is achieved by GitHub Actions, and pipeline provide file `node.yaml` which build project on node 18, and runs tests.
```
Comment (Ewangelina) [inline README.md:25]: I find this sentence hard to understand.
"CI is achieved by GitHub Actions, and the pipeline is provided by the file `node.yaml` which builds the project on node 18, and runs tests."
Or
"CI is achieved by GitHub Actions, and the pipeline provides the file `node.yaml` which builds the project on node 18, and runs tests."

**Review by Ewangelina — CHANGES_REQUESTED** [review]
As it is a public repository, are we sure we want to paste an invite link to trello here?

File: README.md (hunk 2 above)
Comment (kasia251011) [reply README.md:25]: Generally I meant:
`CI is achieved by GitHub Actions, and file node.yaml provides pipeline which builds the project on node 18, and runs tests.`

Would it be okey?

**Comment by kasia251011** [comment]
> As it is a public repository, are we sure we want to paste an invite link to Trello here?

I have blocked option that shares link to Trello with only view access. 😒 I have to link the backlog (requirement), so I think we have to leave it as it is, or do you have another idea?

**Review by Ewangelina — APPROVED** [review]

**Pull request merged**